docs/autobridge.rst

```diff
@@ -0,0 +1,199 @@
+::::::::::::::::::::::::::::
+
+Currently AutoBridge supports the Xilinx Alveo U250 and U280 FPGAs. 
+
+- Feel free to contact us for support of other devices. 
+- Note that the methodology works on large devices (like UltraScale+).
+
+Basic Usage
+::::::::::::
+
+When running ``tapac``, add the ``--enable-floorplan`` option and specify the output constraint file with the ``--floorplan-output`` option.
+
+.. code-block:: shell
+  :emphasize-lines: 5,6
+
+  tapac -o vadd.$platform.hw.xo vadd.cpp \
+    --platform $platform \
+    --top VecAdd \
+    --work-dir vadd.$platform.hw.xo.tapa \
+    --enable-floorplan \
+    --floorplan-output constraint.tcl
+
+AutoBridge needs to know how the AXI interfaces bind to the physical ports, thus the user need to provide a
+`connectivity configuration file <https://docs.xilinx.com/r/en-US/ug1393-vitis-application-acceleration/connectivity-Options>`_ through the ``--connectivity`` option:
+
+.. code-block:: shell
+  :emphasize-lines: 7
+
+  tapac -o vadd.$platform.hw.xo vadd.cpp \
+    --platform $platform \
+    --top VecAdd \
+    --work-dir vadd.$platform.hw.xo.tapa \
+    --enable-floorplan \
+    --floorplan-output constraint.tcl \
+    --connectivity connectivity.ini
+    
+Note
+
+Be careful with the connectivity file. An inferior port binding will cause unnecessary routing congestion. Especially for U280 users, since all the 32 HBM channels are placed on the bottom SLR, you want to balance the usage of channel 0-15 and channel 16-21.
+
+Here is an example of the connectivity file. The syntax is the same as required by the Vitis workflow:
+
+.. code-block:: shell
+
+  [connectivity]
+  sp=VecAdd_1.a:DDR[0]
+  sp=VecAdd_1.b:DDR[1]
+  sp=VecAdd_1.c:DDR[2]
+
+- Note that if you invoke tapac from the command line, you must append an ``_1`` suffix to the top name. On the other hand, you must not append this suffix if you use the cmake flow.
+
+
+By default, AutoBridge uses the resource estimation from HLS report.
+This can be fairly inaccurate (especially for LUT and FF) and affect the QoR.
+TAPA can be configured to use RTL synthesis result for each task instance by the ``--enable-synth-util`` option.
+This option instructs TAPA to run logic synthesis of each task to get a more accurate resource usage, thus improving the quality of the floorplanning results.
+
+.. code-block:: shell
+  :emphasize-lines: 8
+
+  tapac -o vadd.$platform.hw.xo vadd.cpp \
+    --platform $platform \
+    --top VecAdd \
+    --work-dir vadd.$platform.hw.xo.tapa \
+    --enable-floorplan \
+    --floorplan-output constraint.tcl \
+    --connectivity connectivity.ini \
+    --enable-synth-util
+
+.. note::
+
+  Tasks are synthesized in parallel for the post-synthesis area report. Still, this step could takes a while as RTL synthesis is much slower than C synthesis. By default, tapac will invoke 8 parallel Vivado processes to synthesize each task. You could allow more processes through the ``--max-parallel-synth-jobs`` option.
```

Comment:
Typo: "cound takes". Also, maybe mention the memory consumption, which is why we only run 8 in parallel?